lib/resources/Disputes.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+var StripeResource = require('../StripeResource');
+var stripeMethod = StripeResource.method;
+
+module.exports = StripeResource.extend({
+
+  path: 'disputes',
+
+  includeBasic: [
+    'list', 'retrieve', 'update', 'setMetadata', 'getMetadata'
```

Comment:
I think we should be consistent here: all other resources have the `setMetadata` and `getMetadata` methods, so a dispute object should be no different.